CHANGELOG.md

```diff
@@ -48,7 +48,7 @@ These removals were implemented in the following PRs: [#3637](https://github.com
 #### General
 
 - Replaced revert strings and require statements with custom errors. ([#4261](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4261))
-- Bumped minimum compiler version required to 0.8.20 ([#4288](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4288))
+- Bumped minimum compiler version required to 0.8.19 ([#4288](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4288))
```

Comment:
```suggestion
- Bumped minimum compiler version required to 0.8.20 ([#4489](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4489))
```

Reply:
We should reference both PRs in my opinion.

```suggestion
- Bumped minimum compiler version required to 0.8.20 ([#4489](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4489)) ([#4288](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4288))
```